main/modes/platformer/gameData.h

```diff
@@ -6,6 +6,13 @@
 //==============================================================================
 
 #include <stdint.h>
+#include "led_util.h"
+#include "common_typedef.h"
+
+//==============================================================================
+// Constants
+//==============================================================================
+#define NUM_LEDS 8 //Also defined in swadgeMode.h
```

Comment:
Please don't redefine this, just `#include "swadgeMode.h"` here. It's gonna stay as 8 this year, but I don't know for the future, and there's now precedence for porting modes from older swadges. Keep it portable!

Reply:
Fixed